siteapi/v1/tests/test_group.py

```diff
@@ -706,3 +706,48 @@ def test_get_scope_list(self):
 
         res = self.client.get(reverse('siteapi:node_list', args=('g_root', )))
         self.assertEqual(expect, res.json())
+
+    def test_group_user_search(self):
+        role_group = Group.valid_objects.create(uid='group_3', name='group_3', parent=self.root, accept_user=False)
+        role_3 = Group.valid_objects.create(uid='role_3', name='role_3', parent=role_group, order_no=3)
+        user = User.create_user('zhangsan', 'zhangsan')
+        user.name = '张三'
+        user.email = '13412341233@qq.com'
+        user.mobile = '13412341233'
+        user.created = '2019-01-01 00:00'
+        user.last_active_time = '2019-02-01 00:00'
+        user.save()
+        GroupMember.valid_objects.create(user=user, owner=role_3)
+        user2 = User.create_user('zhangsi', 'zhangsi')
+        user2.name = '张四'
+        user2.email = '13412341234@qq.com'
+        user2.mobile = '13412341234'
+        user2.created = '2019-03-01 00:00'
```

Comment:
RuntimeWarning: DateTimeField User.last_active_time received a naive datetime (2019-03-01 00:00:00) while time zone support is active.

Reply:
refacted test time